Sources/PackageLoading/ManifestJSONParser.swift

```diff
@@ -41,49 +51,70 @@ enum ManifestJSONParser {
         identityResolver: IdentityResolver,
         fileSystem: FileSystem
     ) throws -> ManifestJSONParser.Result {
-        let json = try JSON(string: jsonString)
+        let input = try JSONDecoder.makeWithDefaults().decode(Input.self, from: jsonString)
```

Comment:
maybe worth while have a static instance of the json decoder if it thread safe

Reply:
Yah, maybe. I think separate from this PR, though.